hs/src/Reach/AST/DLBase.hs

```diff
@@ -55,12 +55,16 @@ data DLType
   | T_Object (M.Map SLVar DLType)
   | T_Data (M.Map SLVar DLType)
   | T_Struct [(SLVar, DLType)]
+  | T_TokenBalances Integer
```

Comment:
Don't know the size of the array until after evaluation; this will transform to the actual array type

Reply:
What's the integer?